### PR TITLE
chore(rebuild): handle custom blockquotes

### PR DESCRIPTION
**Pull request by montogeek**

### CODE REVIEW

File: tip.js [hunk 1]
```diff
@@ -0,0 +1,76 @@
+// var unified = require('unified');
```
Comment (montogeek) [inline tip.js]: Should also be moved to a subdirectory, see https://github.com/webpack/webpack.js.org/pull/2046#discussion_r181557862
Need clean up, and tests would be cool!

File: tip.js [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+// ?> This is a todo.`;
+
+// const header = `## Header`;
+
+// const md = "T> The `output` property has [many more configurable features](/configuration/output) and if you like to know more about the concepts behind the `output` property, you can [read more in the concepts section](/concepts/output)."
+
+const classMap = {
+  'T>': 'tip',
+  'W>': 'warning',
+  '?>': 'todo'
```
Comment (dhruvdutt) [inline tip.js]: Great work. :ok_hand:

**Comment by skipjack** [comment]
Rebased and refactored slightly to make this a proper `remark` plugin. @montogeek I'm naming these plugins `remark-***` for now and keeping them in the `/src/utilities` directory but you should think about separating them out to repos in your namespace or in the `remarkjs` organization.

> Need clean up, and tests would be cool!

Cleanup done (at least for simple `mapping` feature) but yeah tests would be great. I'd recommend doing that once it's separated to another repo though. The whole point of creating these is so that we can abstract and maintain elsewhere.

We want to keep the codebase here as light and easy to maintain as possible. This allows the site to remain content-focused and allows others to use the same tools we build for it elsewhere. There's probably even more code (e.g. certain components) that can be factored out into shared open-source packages even after the `rebuild` branch is merged.

cc @jeremenichelli

**Comment by montogeek** [comment]
@skipjack I will try to find some time this week to do it, thanks for cleaning it out

**Comment by jeremenichelli** [comment]
It would be super good to turn our remark plugins to packages. We should
reach out if the remark org is interested in hosting them too.
-- 

*Jeremias Menichelli*


*web developer*
skype | jeremias.menichelli
<http://jeremenichelli.github.io>
github | jeremenichelli

**Pull request merged**